Updated the description of span tags and attributes (#26040)
DataDog/documentation: content/en/tracing/trace_explorer/query_syntax.md

There are two types of *terms*:

> * A **Span tag:**  Enrichments of context related to the span. For instance host or container tags describing the infrastructure the service is running on.
> * A **Span attribute:** Content of the span, collected via auto or manual instrumentation in the application.

Reviewer: ```suggestion
* **Span tag**:  Enrichments of context related to the span. For instance, host or container tags describing the infrastructure the service is running on.
* **Span attribute**: Content of the span, collected with automatic or manual instrumentation in the application.
```
